src/app/personside/dialogpanel/fellesStyling.tsx

```diff
@@ -51,23 +51,37 @@ const SpinnerWrapper = styled(FillCenterAndFadeIn)`
 export function DialogpanelFeilmelding() {
     return <AlertStripeFeil>Det skjedde en feil ved sending av melding</AlertStripeFeil>;
 }
-function MeldingSendtVerktoyLinje(props: { fritekst: string; sisteTraad: Traad }) {
+
+function useSentMelding(fritekst: string) {
```

Comment:
ser  veldig bra ut 👍 Kan du legge på en sjekk som sjekker at sisteSendte melding også er sendt for maks f.eks ti min siden? Da får vi litt mer trygghet på at det ikke kan skje en rar feil med fritekst-sammenligningen her.